applenav/src/components/Products/ProductsWrapper.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { NavLink, Link, Route } from "react-router-dom";
+import "./Products.css";
+
+// import Products from "./Products";
+import Mac from "./Mac";
+import Ipad from "./iPad";
+import Iphone from "./iPhone";
+import Watch from "./Watch";
+import Tv from "./TV";
+import Music from "./Music";
+
+const navLinks = ["Mac", "iPad", "iPhone", "Watch", "TV", "Music", "Support"];
+
+const ProductsWrapper = props => {
+  return (
+    <div>
+      <Route path="/Mac" component={Mac} />
```

Comment:
Missing root component
```suggestion
      <Route exact path='/' component={Home} />
      <Route path="/Mac" component={Mac} />
```